tests/integration/lib/instance.rb

```diff
@@ -38,6 +38,15 @@ def run_konnector(slug, account_id)
     @stack.run_konnector self, slug, account_id
   end
 
+  def run_job(type, args)
+    @stack.run_job self, type, args
+  end
+
+  def db_prefix()
```

Comment:
[nit] You can omit `()` here

Reply:
I prefer to be explicit with parentheses, I really dislike this ruby feature. Anyway, I removed the dead code, so no problem anymore :)